sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  QueueOptions,
+  TopicOptions,
+  SubscriptionOptions,
+  ServiceBusAtomManagementClient
+} from "../../src";
+
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
+
+const env = getEnvVars();
+let client: ServiceBusAtomManagementClient;
+
+async function getManagementClient() {
+  if (client == undefined) {
+    client = await new ServiceBusAtomManagementClient(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]);
+  }
+  return client;
+}
+
+export async function recreateQueue(queueName: string, parameters: QueueOptions): Promise<void> {
+  client = await getManagementClient();
```

Comment:
Since `getManagementClient()` already is setting the `client` variable, do we have to set the same variable again here?